src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -141,13 +142,13 @@ class BulletRigidObject : public RigidObject {
    * See @ref btCollisionObject::isActive.
    * @return true if active, false otherwise.
    */
-  bool isActive();
+  bool isActive() override;
 
   /**
    * @brief Set an object as being actively simulated rather than sleeping.
    * See @ref btCollisionObject::activate.
    */
-  void setActive();
+  void setActive() override;
```

Comment:
This added `override` is unrelated to this now being a subclass of MotionState, right?

Reply:
Correct, this just just another function which is virtual in PhysicsManager but was not explicitly marked as override in BulletPhysicsManager.